articles/Smog.xml

```diff
@@ -0,0 +1,78 @@
+            <p>City borders could not restrict smog; farmers in the mid-1940s noticed that their crops were perishing from the toxic gas. As a result, many farmers lost money because of the destroyed crops. Some farmers stopped harvesting altogether. In one account, a “grower suffered $30,000 in dead crops within twelve hours of the vapors’ arrival” (Jacobs and Kelly 50). More reports like this became very common and “by 1954, experts estimated the loss to Southland crops at $3 million” (Jacobs and Kelly 52). Oil companies also caused pollution, which would prove to be an especially difficult situation to manage; with a “region whose population nearly doubles in ten years”, Los Angeles needed to produce more oil. But, more oil would mean more emissions. Air pollution was clearly jeopardizing the future of Southern California.</p>
+            <p>Ronald Reagan governed the state of California in the 1970s with relaxed policies on regulating pollution control. His pro-business strategy was widely popular, and was part of a general political trend throughout the state that favored the unrestricted free market as opposed to governing bodies. Jacobs and Kelly describe Reagan as being inattentive when dealing with environmental agency authorities, often changing the subject when environmental protection authorities tried to address the growing pollution crisis. Following Reagan’s term as governor, there was a shift in political authority. Governor Jerry Brown prioritized pollution control to make good on his campaign promise to “restore blue skies over California”. This dynamic political shift over the period of these two administrations caused tension within the state government. The auto industry corporations, which were unrestricted under Reagan, were now forced to include basic pollution control devices in all newly manufactured vehicles. Retrofit pollution devices and catalytic converters were new requirements for all cars manufactured and sold. As a result, Los Angeles was able to mitigate the smog problem without greatly compromising the profitability of the automobile and oil industries (Jacobs and Kelley 75-77).</p>
+         </div>
+      </body>
+      
+      <back>
+         
+         <note type="mapmarker"><!-- this will be the title & text in your map marker pop-up -->
+            <placeName type="environment and science">Smog in Los Angeles, UCLA Young Reserarch Library Special Collections</placeName>
```

Comment:
We feel that UCLA is a valid location because the author got much of the information to right the book from here. The author write an inscription thanking UCLA for all its help with the book. Also, UCLA was mentioned many times throughout.